src/modules/auth/hooks/useNonceSync.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState, useCallback } from 'react';
+import { useQueryClient } from '@tanstack/react-query';
+import { useCurrentApplication } from '@blockchainApplication/manage/hooks';
+import { useCurrentAccount, useAccounts } from '@account/hooks';
+import useSettings from '@settings/hooks/useSettings';
+import { AUTH } from 'src/const/queries';
+import { useAuthConfig } from './queries';
+
+// eslint-disable-next-line max-statements
+const useNonceSync = () => {
+  const queryClient = useQueryClient();
```

Comment:
Is there a way to group these vars with space separators, to scope connected logic. For example 

```
 const [currentAccount] = useCurrentAccount();
  const currentAccountAddress = currentAccount.metadata.address;
  const { mainChainNetwork } = useSettings('mainChainNetwork');
  const queryClient = useQueryClient();
  const chainID = currentApplication.chainID;
  const customConfig = {
    params: {
      address: currentAccountAddress,
    },
  };
  const baseUrl = mainChainNetwork?.serviceUrl;
  const config = useAuthConfig(customConfig);
  const authData = queryClient.getQueryData([AUTH, chainID, config, baseUrl]);
  const onChainNonce = authData?.data.nonce;

const [accountNonce, setAccountNonce] = useState(onChainNonce);

const currentAccountNonce = getNonceByAccount(currentAccountAddress);
  ```

Reply:
Ah, i think that is what Manu is suggesting in https://github.com/LiskHQ/lisk-desktop/pull/5390/files#r1365008159